src/client/Microsoft.Identity.Client/PlatformsCommon/Shared/NullDeviceAuthManager.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Http;
+using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
+
+namespace Microsoft.Identity.Client.PlatformsCommon.Shared
+{
+    internal class NullDeviceAuthManager : IDeviceAuthManager
```

Comment:
`DeviceAuthHelper.GetBypassChallengeResponse(response)` is still a response, but CanHandleDeviceAuthChallenge  is set to false?

Reply:
removed the CanHandleDeviceAuthChallenge  as it is no longer needed